unit_tests/src/test_process_stash.cpp

```diff
@@ -53,15 +53,22 @@ TEST_CASE("process_stash basic operations")
     CHECK(boost::any_cast<pod&>(*p).y == a_pod.y);
 
     // Show that the handles can be used to erase data in the stash.
-    irods::process_stash::erase(h1);
+    auto was_removed{irods::process_stash::erase(h1)};
+    CHECK(was_removed);
```

Comment:
Could the `erase` call be put directly into `CHECK`?

Reply:
I had the same thought :-)

Reply:
Can remove the `was_removed`. 👍